### DISCUSSION TITLE
LTMEAT Flood

### DISCUSSION THREAD

**Discussion by twentyzeven** [3kh0/ext-remover#884] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a way I can fill the folder with the pages faster? Or is it manual and I have to paste in every individual page?

**Comment by AshtonDavies** [comment, marked as answer]
If your key repeat is too slow to paste 800+ tabs in about a minute in the bookmark manager, you can paste 10 or so and highlight all of it and paste a larger amount of bookmarks. It shouldn't take very long though.

Update: You can automate the setup process on my [official details page](https://chrose.pages.dev/detail/LTMEAT%20Flood).
